src/app/client/src/app/modules/dashboard/components/data-chart/data-chart.component.ts

```diff
@@ -230,6 +230,9 @@ export class DataChartComponent implements OnInit, OnDestroy {
     if (_.get(this.chartConfig, 'labels')) {
       labels = _.get(this.chartConfig, 'labels');
     }
+    _.forEach(labels, (label, key) => {
+      labels[key] = _.capitalize(label);
```

Comment:
This looks good, but you need to do the same when showing in filters also.
There also labels should in be title case.

Reply:
That is already fixed and PR is merged

Reply:
oh okay. Then it's perfect.